std_msgs/String item
---
std_msgs/Header header
geometry_msgs/Point point
bool found